counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
```

Comment:
I would prefer to reject it to remove the ambiguity (in both the code and the spec) as well as simplify the code. The only place where it might be ok is if the longname segment (e.g. `;;lPARENTASSET.mychild`) was totally missing from the description, and in that case I'd still probably want to reject it (but it *might* work to just have counterparty-lib deal with it in that case).

Really, the only place being more strict like this would affect would be with the 3rd party library writers (e.g the JS counterparty library, etc), as anyone else will be using the counterparty-lib API, and they will just specify "asset" as the full longname, and description to whatever, and not even know about `;;l`. I have no problem making library authors jump through an additional (minor) hoop if it means more clear cut logic in both the spec and implementation. :)